common/services/move.js

```diff
@@ -34,6 +49,43 @@ const moveService = {
     page = 1,
     isAggregation = false,
   } = {}) {
+    // TODO: This is more of a temporary solution to solve the problem where
+    // the API doesn't have a concept of what locations a user has access to
+    //
+    // Once Auth is moved to the API we would be able to remove this as the API
+    // would know to only return moves that a user has access to
+    if (
```

Comment:
You might find this verbose and/or indirect

```js
    const getMultipleRequestsPromise = (direction) => {
      if (
        (filter[`filter[${direction}_location_id]`] || '').split(',').length >
        LOCATIONS_BATCH_SIZE
      ) {
        return makeMultipleRequests(
          moveService.getAll,
          {
            filter,
            combinedData,
            page,
            isAggregation,
          },
          `filter["filter[${direction}_location_id]"]`
        )
      }
    }

    const fromPromise = getMultipleRequestsPromise('from')
    if (fromPromise) { return fromPromise }

    const toPromise = getMultipleRequestsPromise('to')
    if (toPromise) { return toPromise }
``

Reply:
Hmm, actually there might be a nice middleground using that approach to move the check to `makeMultipleRequests` instead...let me have a try...

Reply:
Have updated it a bit so the initial function is a little easier to read. Let me know what you think.